### ISSUE TITLE
Export peers information from a node

### ISSUE THREAD

**Issue by adambabik** [status-im/status-go#522]
Acceptance Criteria:
* use `expvar` to expose metrics,
* expose number of peers,
* expose a list of peers with information like peer ID, IP and so on.

**Comment by adambabik** [comment]
As discussed with @divan, the scope of this issue must be extended to also support metrics in `statusd` as we want to debug how a mobile node connects to the Whisper cluster.

**Comment by divan** [comment]
Right, peer information comes from the p2p layer, so it's useful both for whisper- and les-enabled status nodes.

**Comment by pablochacin** [comment]
Hi I'm new to status project. I've looking for some concrete issues to help me get familiar with the code and I think this looks promising. I will appreciate any suggestion on how to approach the issue.

**Comment by andytudhope** [comment]
Hey @pablochacin! That's really awesome, but I think that Adam and Ivan have already done most of the worked and are just blocked by one other thing before merging - see [here](https://github.com/status-im/status-go/pull/523). @adambabik would you confirm that and perhaps - if possible - suggest a different issue @pablochacin could whet his appetite with?

**Comment by pablochacin** [comment]
Thanks, that would be awesome.

**Comment by adambabik** [comment]
It was done in #558
